mmv1/third_party/terraform/resources/resource_bigtable_gc_policy.go

```diff
@@ -84,12 +89,33 @@ func resourceBigtableGCPolicy() *schema.Resource {
 				Description: `The name of the column family.`,
 			},
 
+			"gc_rules": {
```

Comment:
If this field is not updatable, we probably need to add `ForceNew: true` to it

Reply:
I am pretty sure column families is updatable. That said, we need to have a test to verify this.

Reply:
@shuyama1 Is there an example on how to add `update` test?